libs/tree-sitter-wing/grammar.js

```diff
@@ -266,6 +267,15 @@ module.exports = grammar({
     break_statement: ($) => seq("break", $._semicolon),
 
     continue_statement: ($) => seq("continue", $._semicolon),
+    
+    if_let_statement: ($) => seq(
+      "if let",
```

Comment:
Optional idea for a future enhancement...
```suggestion
      // TODO: support "if let var"
      "if let",
```